draft-ietf-tls-tls13.md

```diff
@@ -1383,10 +1383,7 @@ Structure of this message:
 
 %%% Key Exchange Messages
 
-       struct {
-           uint8 major;
-           uint8 minor;
-       } ProtocolVersion;
+       uint8 ProtocolVersionID[2];
```

Comment:
On Friday, September 09, 2016 01:39:30 am Martin Thomson wrote:
> I don't see the point of this being `ProtocolVersionID` over just `ProtocolVersion`.

It's not a version number, but rather an arbitrary ID for that version number. "1.3" is the version number and "{3,4}" is the ID.